### PR TITLE
NSA-2766 Filters out goverment orgs and department for education

### PR DESCRIPTION
**Pull request by arshedmust**

### CODE REVIEW

File: src/app/organisations/data/organisationsStorage.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ const getOrgById = async (id) => {
   return org;
 };
 
-const pagedSearch = async (criteria, pageNumber = 1, pageSize = 25, filterCategories = undefined, filterStates = undefined) => {
+const pagedSearch = async (criteria, pageNumber = 1, pageSize = 25, filterCategories = undefined, filterStates = undefined, filterOutOrgNames = undefined) => {
```
Comment (DrogoNevets) [inline src/app/organisations/data/organisationsStorage.js]: if filterOutOrgNames is defaulted to [] (see next comment)

File: src/app/organisations/data/organisationsStorage.js [hunk 2]
```diff
@@ -186,6 +186,14 @@ const pagedSearch = async (criteria, pageNumber = 1, pageSize = 25, filterCatego
       },
     };
   }
+  
+  if (filterOutOrgNames && filterOutOrgNames.length > 0) {
```
Comment (DrogoNevets) [inline src/app/organisations/data/organisationsStorage.js]: this can be sped up up just by doing the length test

Comment (arshedmust) [reply src/app/organisations/data/organisationsStorage.js]: Done

File: src/app/organisations/data/organisationsStorage.js (hunk 1 above)
Comment (arshedmust) [reply src/app/organisations/data/organisationsStorage.js]: Done

**Pull request merged**